onward/app/views/feedback.scala.html

```diff
@@ -25,6 +25,7 @@ <h1 itemprop="headline" class="content__headline">Solve a technical issue</h1>
                                             <li>crashing</li>
                                             <li>reloading</li>
                                             <li>very slow site access</li>
+                                            <li>display ads problem</li>
```

Comment:
I'm not sure if this is the right phrasing? I would say `issues with advertising`. I'm not sure if 'display ads' might be too specialised vocabulary for the average punter.